src/components/Registration/VolunteerSignupForm.vue

```diff
@@ -113,6 +131,7 @@
           <td colspan="2">
             <input
               class="form-control"
+              v-bind:class="{'form-control-invalid': invalidInputs.indexOf('favoriteAcademicSubject') > -1}"
```

Comment:
Later on, we should make something like a `form-field` component to cut down on the repetition of logic. But for now, looks good :)